test/config/config.go

```diff
@@ -90,6 +96,8 @@ func (c *CiliumTestConfigType) ParseFlags() {
 		"Specifies timeout for test run")
 	flagset.StringVar(&c.Kubeconfig, "cilium.kubeconfig", "",
 		"Kubeconfig to be used for k8s tests")
+	flagset.StringVar(&c.RegistryCredentials, "cilium.registryCredentials", "",
```

Comment:
nit: I think there's a precedence for kebab case (`registry-credentials`)

Reply:
All the other args are camel case, no?
cf: https://github.com/cilium/cilium/blob/7b846fd64a541b30b960f05efde454f237e2b948/test/config/config.go#L69

Reply:
Ah, for some reason I only remembered `operator-image`, `operator-tag`, etc.